### PR TITLE
Add common labels

### PR DESCRIPTION
**Pull request by uzzz**
Some organisations (like mine) require certain labels to be present on every Kubernetes resource, for example, a resource ownership label (which team owns a resource).

This PR adds `commonLabels` option that adds labels to every resource to make this use case possible.

### CODE REVIEW

**Comment by uzzz** [comment]
@cla-bot check

**Comment by uzzz** [comment]
@cla-bot check

**Comment by mosabua** [comment]
Did you submit a CLA yet? If yes, we just have to wait for it to be filed before merge. If not, please do so.

**Comment by uzzz** [comment]
@mosabua I did sign and sent it to cla@trino.io. Not sure, maybe I did something not quite correct and the bot can't process my CLA

**Comment by mosabua** [comment]
@uzzz we have CLA processing on hold until August ... stay tuned for that.

**Comment by uzzz** [comment]
@cla-bot check

**Review by mosabua — APPROVED** [review]
This looks good to me.

**Comment by mosabua** [comment]
Any concerns @losipiuk

**Comment by mosabua** [comment]
Could you squash the commits please @uzzz ?

**Comment by mosabua** [comment]
All good .. I was able to squash and push.

**Pull request merged**
